apps/web/app/routes/examples/forms/form-with-radio-children.tsx

```diff
@@ -0,0 +1,94 @@
+import hljs from 'highlight.js/lib/common'
```

Comment:
What if we used the `howYouFoundOutAboutUs` field from the existing `form-with-children` example for that? The use case for select on children is already tested in the `field-layout` example, so I think it would be fine.